docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -446,6 +447,8 @@ Ansible does apply variable precedence, and you might have a use for it. Here is
   #. include params
   #. extra vars (for example, ``-e "user=my_user"``)(always win precedence)
 
+If inventory file is located in the same directory as a playbook, adjacent group_vars/ are interpreted twice both as inventory group_vars/ and playbook group_vars/, therefore, getting precedence of the playbook group_vars.
```

Comment:
This is incorrect, at no point does the inventory file get precedence over the playbook's group_vars.

What happens is that since the file is processed x2, it ends up with the highest precedence, that of the playbook relative file.

Reply:
Hello, thank you for suggestion I accept it, but correct for playbook been 'stronger'.

Reply:
It was added by Don Naro